R/pkg/R/DataFrame.R

```diff
@@ -3175,7 +3176,8 @@ setMethod("with",
 #' @aliases str,SparkDataFrame-method
 #' @family SparkDataFrame functions
 #' @param object a SparkDataFrame
-#' @examples \dontrun{
+#' @examples
+#' \dontrun{
```

Comment:
There are a lot of these in `functions.R`. Maybe you could fix them too with this PR?